docs/architecture.md

```diff
@@ -0,0 +1,52 @@
+# Architecture
+
+Newman Dashboard is a real-time reporter for newman to control, view and debug runs on a WebUI dashboard.
+
+## Pub - Sub
+
+We establish a pub/sub architecture between the newman runs and the WebUI(frontend) through a central server known as the broker. <br/>
```

Comment:
I don't really like the term `broker`. Let's call it the `daemon` and one of the responsibilities of the `daemon` would be to _act_ as a broker.

Reply:
I think we missed this... 😅

Reply:
Hey! Sorry I just wanted to ask that do you want me to replace all the occurrences of `broker` with `daemon` or just the one here?

Reply:
All if we can. 😅

Reply:
Done!